src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
From your second link:
>In at least some cases, the postfix operator will be less efficient.

>However, in 99.99% cases, it won't matter because (a) it'll be acting on a simple or primitive type anyway and it's only a problem if it's copying a big object (b) it won't be in a performance critical part of code (c) you don't know if the compiler will optimise it or not, it may do.

Sounds to me like it's something that compiler should optimize out… If it doesn't already, then *the compiler should be improved*.

IMHO postfix is way clearer and easier to understand than prefix.

And given that  `cleaner code` > `theoretically faster but harder to understand code` …